workflow/installertest.groovy

```diff
@@ -92,41 +205,79 @@ def runShellTest(String imageName, def shellCommands, def stepNames=null) {
           // Second, the sh workflow step often will use the default posix shell
           // The default posix shell does not support pipefail, so we have to invoke bash to get it
           
-          String argument = 'bash -c \'set -o pipefail; '+cmd+" | tee \"testresults/$fileName-$name"+'.log'+'" \''
+          String argument = cmd+" | tee \"testresults/${fileName}-${name}.log\""
```

Comment:
Yeah, probably - I started redoing the string editing without remembering the first part. Fixing!